### DISCUSSION TITLE
how i can use the bulk registration

### DISCUSSION THREAD

**Discussion by Sephiroth80** [poweradmin/poweradmin#595] [General]
i was read lot of massages but i can understand how i can use it to charge multiple registry
for example if i have a declarete domain and i want to charge 10 registry A
subdmain.domain.com  IN A XXX.XXX.XXX
but when i do that i have errors like
Error: You have invalid characters in your hostname.
cliente.xxx.com.ar in ns a1-31.akam.net - Invalid hostname.
for examply.
thanks a lot

**Comment by edmondas** [comment]
I'm sorry, it's really hard to understand what kind of problem you're having. In any case, if it's a hostname-related error, the given input for the selected DNS record type is wrong, please check the host name that you provided if it's correct.

From another point of view, bulk registration means that you can use a selected template and enter multiple domains in a text box. So maybe your template isn't valid, there's no proper validation at the moment, so you need to be more careful about putting in data and placeholders.

**Comment by edmondas** [comment]
> i was read lot of massages but i can understand how i can use it to charge multiple registry for example if i have a declarete domain and i want to charge 10 registry A subdmain.domain.com IN A XXX.XXX.XXX but when i do that i have errors like Error: You have invalid characters in your hostname. cliente.xxx.com.ar in ns a1-31.akam.net - Invalid hostname. for examply. thanks a lot

The bug affecting bulk record updates in the edit view has been fixed in the latest master branch. Please update your code to access this fix.
